src/OceanSimulations/OceanSimulations.jl

```diff
@@ -96,10 +104,12 @@ function ocean_simulation(grid; Δt = 5minutes,
         momentum_advection = nothing
     end
 
```

Comment:
we probably need to make sure that T and S exist if we are passing tracers as an argument. Something like this might work
```suggestion
tracers = unique(tuple(tracers..., :T, :S))
```

Reply:
Personally I think it is more obvious to simply check whether `:T` and `S` are there. Plus lets all remember not to try to cram lots of code onto one line.

Reply:
The model constructor will show an error, right? Letting the model constructor error is consistent with the approach we are also taking for biogeochemical tracers.

The discussion is whether we should add tracers automatically, or require that users add tracers explicitly.

Reply:
Well, at this moment it's a mix that will not work because in the boundary conditions it's implied that T and S exist. The approach we used is to add T and S (and e if using CATKE) automatically. The question is if we want to change that behavior or not. I would vote for no, but even if we do it, better to do that in another PR.

Reply:
The default is still `tracers = (:T, :S)`.

Reply:
Of course, it is, but there is the possibility to pass tracers that do not include T and S, which is a fundamental change with respect to how we defined this function. 
There are pros and cons to that, but for the moment, `ocean_simulation` defines boundary conditions for T and S whether you pass T and S or not. It does not really make sense to do so if we allow not having T and S in an ocean simulation. So I would say, let's keep the intended behavior for now and we can discuss this in an issue.

Reply:
Thanks, I wasn't really sure what was the best way to handle the T, S tracers. So for now I can add the additional line @simone-silvestri suggested to ensure that T and S exist.

> we probably need to make sure that T and S exist if we are passing tracers as an argument. Something like this might work

`tracers = unique(tuple(tracers..., :T, :S))`

I'll just add that from my user point of view, with a `tracer` kwarg I would assume that all tracers need to be defined including T and S, it would be a bit odd to not define those two unless the kwarg was `bgc_tracers` or something else.

Reply:
right, we are trying to disentangle two objectives:
- being explicit about the arguments that go into the model
- Avoid awkward situations like having to explicitly declare T and S when adding a biogeochemistry model and not when the latter is not added. 
This is apparently not the correct way and we probably need to figure out a better solution that satisfies both these requirements. Let's discuss solutions to this problem in an issue.

Reply:
I disagree that it's awkward to explicitly declare T, S. What do others think?